platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -3,9 +3,13 @@
 <WixLocalization xmlns="http://wixtoolset.org/schemas/v4/wxl" Culture="en-US">
   <String Id="DowngradeError" Value="A newer version of [ProductName] is already installed." />
   <String Id="Bld_ProductName" Value="Swift Build Tools" />
+  <String Id="BldNoAsserts_ProductName" Value="Swift Build Tools (Asserts disabled)" />
   <String Id="Cli_ProductName" Value="Swift Command Line Tools" />
+  <String Id="CliNoAsserts_ProductName" Value="Swift Command Line Tools (Asserts disabled)" />
   <String Id="Dbg_ProductName" Value="Swift Debugging Tools" />
+  <String Id="DbgNoAsserts_ProductName" Value="Swift Debugging Tools (Asserts disabled)" />
   <String Id="Ide_ProductName" Value="Swift IDE Integration Tools" />
+  <String Id="IdeNoAsserts_ProductName" Value="Swift IDE Integration Tools (Asserts disabled)" />
```

Comment:
Similarly, we should use the variant spelling and update the default ones to indicate the variant as well.

Reply:
changed